openedx/core/lib/request_utils.py

```diff
@@ -141,6 +145,10 @@ def process_request(self, request):
         - TOP_N_COOKIE_GROUPS_CAPTURED
 
         """
+
+        if request.META.get('HTTP_COOKIE', None):
+            set_custom_attribute('cookies.header.size', len(request.META['HTTP_COOKIE'].encode('utf-8')))
```

Comment:
Out of curiosity, why the `encode` call? I guess that's measure bytes rather than characters?

Reply:
Yep.